### PR TITLE
uptime for both of the peers.

### PR DESCRIPTION
**Pull request by hthayko**

### CODE REVIEW

File: src/lib/utility.js [hunk 1]
```diff
@@ -60,9 +62,15 @@ module.exports = {
         } else {
           rippleds[p_pk] = {ipp: p_ipp, version: p_v, uptime: uptime};
         }
-
       });
     });
+    // correcting uptime once rippleds and maxUptimeByIpp are ready
+    _.each(rippleds, function(r) {
+      var peerMaxUptime = r.ipp && maxUptimeByIpp[r.ipp];
```
Comment (wilsonianb) [inline src/lib/utility.js:69]: Does this assignment work?

Comment (hthayko) [inline src/lib/utility.js:69]: yup, it's a guard operator :) usually if I have a deeply nested property, i do
var value = foo && foo.bar && foo.bar.value, 
to guard against undefined's

Comment (hthayko) [inline src/lib/utility.js:69]: BTW, this code is tested, and I posted a screenshot in  crawler channel

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**